packages/flutter_tools/lib/src/android/android_builder.dart

```diff
@@ -39,4 +39,6 @@ abstract class AndroidBuilder {
     bool deferredComponentsEnabled = false,
     bool configOnly = false,
   });
+
+  Future<List<String>> getBuildOptions({required FlutterProject project});
```

Comment:
I am not sure if on this function is the right place but please add some documentation for what "getBuildOptions" is for an who should use it.